batch/sql/jobs-add-n-max-attempts.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE jobs ADD COLUMN n_max_attempts INT NOT NULL DEFAULT 404, ALGORITHM=INSTANT;
```

Comment:
Why is this default 404?

Reply:
Oops, this was a placeholder while testing this out. This is slightly different semantically from the defaults in the code (this is a value to assign any pre-existing batches - which had de-facto infinite retries - instead of a default going forward), but for the sake of consistency, I think changing this to 20 as well is probably most sensible